terracumber_config/tf_files/SUSEManager-4.2-PRV.tf

```diff
@@ -189,8 +189,14 @@ module "cucumber_testsuite" {
     kvm-host = {
       image = "sles15sp3o"
       additional_grains = {
-        hvm_disk_image = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2"
-        hvm_disk_image_hash = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256"
+        hvm_disk_image = {
+          leap = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2"
+          sles = "https://download.suse.de/install/SLE-15-SP4-Minimal-GM/SLES15-SP4-Minimal-VM.x86_64-OpenStack-Cloud-GM.qcow2"
+        }
+        hvm_disk_image_hash = {
+          leap = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256"
+          sles = "1215677036ea32bf2a8d99770b28cc0e76956def0c14d7a8d30f745ec1f0438c"
```

Comment:
Would be better to have the path to a file containing the checksum

Reply:
That was my initial though, too. Sadly we do not have the checksum in its own file but combined with others here: https://download.suse.de/install/SLE-15-SP4-Minimal-GM/SHA256SUMS
I will ask Eric if it is possible to copy the checksum to the mirror, too.

Reply:
We already do include hashes for images on the mirror but every image has its own hash file which is not the case with the image I use. I will adjust one of our scripts on the mirror to be able to extract the hashes to own files so we can use the upstream image/hash and do not have to hard code things.

Reply:
Added with: https://gitlab.suse.de/galaxy/infrastructure/-/merge_requests/659
I will adjust this PR once the MR got merged.